### ISSUE TITLE
Incorrect management of `+` in filenames.

### ISSUE THREAD

**Issue by bleuse** [jiffyclub/snakeviz#107]
I usually name my profiling files using the output `date -Iminutes` in the filename. For example, a file may be named `2018-02-14T07:04+01:00-example.cprof`.

However, snakeviz fails to open the trace file, and fails with the following trace:
```
Traceback (most recent call last):
  File "/home/user/devel/venv36/lib/python3.6/site-packages/snakeviz/main.py", line 28, in get
    s = Stats(profile_name)
  File "/usr/lib/python3.6/pstats.py", line 72, in __init__
    self.init(arg)
  File "/usr/lib/python3.6/pstats.py", line 86, in init
    self.load_stats(arg)
  File "/usr/lib/python3.6/pstats.py", line 99, in load_stats
    with open(arg, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/snakeviz/2018-02-14T07:04 01:00_example.cprof'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/user/devel/venv36/lib/python3.6/site-packages/tornado/web.py", line 1510, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/home/user/devel/venv36/lib/python3.6/site-packages/snakeviz/main.py", line 30, in get
    raise RuntimeError('Could not read %s.' % profile_name)
RuntimeError: Could not read /tmp/snakeviz/2018-02-14T07:04 01:00_example.cprof.
```

It looks like the `+` in the filename is not correctly handled.

**Comment by jiffyclub** [comment]
Interesting, thanks for this report! A `+` is how spaces are encoded in URLs so I can see how snakeviz is ending up looking for the filename with a space instead of a `+`. I'll have to research if there's a good way to handle this.

**Comment by jiffyclub** [comment]
Thanks again for reporting, I think this should be fixed in 6851ee7.

**Comment by bleuse** [comment]
Your fix indeed works for this use case. Thanks for this nifty tool!
